appveyor.yml

```diff
@@ -0,0 +1,21 @@
+branches:
+  only:
+    - dev
+    - sschen/ci
```

Comment:
I've now turned on the setting for "build only if appveyor.yml is in VCS", so no, this can be removed.